lib/MumbleClient.js

```diff
@@ -533,6 +533,16 @@ MumbleClient.prototype._userRemove = function( data ) {
         user._detach();
         delete this.users[ user.id ];
```

Comment:
While you're touching this function, could you do a console.log on `this.users` or something similar to check just _what_ `this.users` is. I have a feeling it's a function on the prototype and this delete statement makes no sense.

If `this.users` _is_ a function, you should also remove this line. It's essentially attempting to do the same your loop does now, but is poking the wrong field and assumes the field is an object indexed by the user id instead of a list.

Reply:
As you expected, it was a function. I've removed this line.